brightray/browser/browser_main_parts.cc

```diff
@@ -89,6 +89,18 @@ void OverrideLinuxAppDataPath() {
   PathService::Override(DIR_APP_DATA, path);
 }
 
+// void OverrideWinAppLogsPath() {
+//   base::FilePath path;
+//   // should be in c:\program files\myapp\logs ?
```

Comment:
Wow... I just clicked the link in my notifications and went to the diff 😕

weird 😆